solutions/standard/variables.tf

```diff
@@ -160,6 +160,68 @@ variable "tags" {
   default     = []
 }
 
+variable "powervs_custom_images" {
+  description = "Optionally import up to three custom images from Cloud Object Storage into PowerVS workspace. image_name: string, must be unique. Name of image inside PowerVS workspace. file_name: string, object key of image inside COS bucket. storage_tier: string, storage tier which image will be stored in after import. Supported values: tier0, tier1, tier3, tier5k. sap_type: optional string, Supported values: null, Hana, Netweaver, use null for non-SAP image."
```

Comment:
Requires 'powervs_custom_image_cos_configuration' and 'powervs_custom_image_cos_service_credentials' to be set.

Reply:
Changed it to ```Requires 'powervs_custom_image_cos_configuration' to be set.```

`powervs_custom_image_cos_service_credentials` is only required if `bucket_access` is set to `private` and that's documented inside `powervs_custom_image_cos_service_credentials`